easybuild/easyconfigs/l/LSMS/LSMS-3_rev237-foss-2016a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'LSMS'
+version = '3_rev237'
+
+homepage = 'https://asc.llnl.gov/CORAL-benchmarks/#lsms'
+description = "LSMS benchmark, part of CORAL suite"
+
+toolchain = {'name': 'foss', 'version': '2016a' }
+
+source_urls = ['https://asc.llnl.gov/CORAL-benchmarks/Science/']
+sources = ['%(name)s_%(version)s.tar.bz2']
+
+checksums = ['8c864c9223b93488663815557e61cd0b']
+
+patches = ['LSMS-3_rev237_fix-Makefiles.patch']
```

Comment:
@foxik0070 please avoid hardcoding the version, use the `%(version)s` template instead:

```python
patches = ['LSMS-%(version)s_fix-Makefiles.patch']
```